dojson/contrib/to_marc21/model.py

```diff
@@ -50,5 +52,15 @@ def do(self, blob):
 
         return output
 
+    def _get_indicators(self, value):
+        """Get indicators from the value."""
+        if isinstance(value, MutableMapping):
+            return value.pop('$ind1', '_'), value.pop('$ind2', '_')
+        if isinstance(value, MutableSequence):
+            return [(v.pop('$ind1', '_'), v.pop('$ind2', '_'))
+                    for v in value][0]
```

Comment:
I can't put it like that because every dictionary inside the list will potentially contain the two keys